app/src/main/java/org/simple/clinic/editpatient/EditPatientUpdate.kt

```diff
@@ -56,6 +56,7 @@ class EditPatientUpdate(
       is AlternativeIdChanged -> next(model.updateAlternativeId(event.alternativeId))
       is BpPassportsFetched -> dispatch(DisplayBpPassportsEffect(event.bpPasssports))
       is InputFieldsLoaded -> dispatch(SetupUi(event.inputFields) as EditPatientEffect)
+      is ColonyOrVillagesFetched -> next(model.updateColonyOrVillagesList(event.colonyOrVillages))
```

Comment:
Currently, it seems like you're combining both the effect handler and update functionality into a single commit. Can we separate them into different commits, going forward?